Cecilifier.Web/wwwroot/js/cecilifier.js

```diff
@@ -289,6 +289,7 @@ function initializeSettings(formattingSettingsSample) {
         new Setting(ElementKind.Delegate, {line: startLine + 30, ch: 5}, "Delegate", "delegate prefix","ADelegate", "del"),
         new Setting(ElementKind.LocalVariable, {line: startLine + 32, ch: 5}, "Local Variable", "local variable prefix","i", "lv"),
         new Setting(ElementKind.MemberReference, {line: startLine + 34, ch: 5}, "Member Reference", "Member reference prefix","i", "mr"),
+        new Setting(ElementKind.Label, {line: startLine + 36, ch: 5}, "Jump Label", "Jump Label Prefix","jump", "lbl"),
```

Comment:
Thanks for the PR

This is the correct fix. It is just missing an update ([in this exact same file](https://github.com/adrianoc/cecilifier/blob/d292a8455b4df2d8ee1734000964be9ed34a6d4c/Cecilifier.Web/wwwroot/js/cecilifier.js#L138)) to the example (showed when you are configuring the prefixes for each element) 

Let me know if you need any help with that.

Reply:
Completely unrelated, but I confess that I confess I don't always open PRs on issues I report, but I'm excited about Cecil lately. 

I added an example string in the location you linked. I hope it's okay.